src/test/java/org/openstreetmap/atlas/utilities/filters/AtlasEntityPolygonsFilterTest.java

```diff
@@ -99,6 +100,37 @@ public boolean polygonEntityIntersecting(final Polygon polygon, final AtlasEntit
                 return false;
             }
         }
+
+        // note this is the only one now used by the filter
```

Comment:
What does this comment mean?

Reply:
The geometricSrufaceEntityIntersecting is the only function that is being used now. Which relates to my overall PR question of should I get rid of the Intersection Policy Interface and replace it with a customizable BiPredicate of AtlasEntity and GeometricSurface that I cast to serializable?

Reply:
Good question. Personally I think having a BiPredicate of something like that can get fairly messy and abstracting it away as a policy makes more sense. Open to other ideas though :)